ang/workflow/directives/workflow-list.directive.html

```diff
@@ -17,7 +17,7 @@ <h1 crm-page-title>{{:: ts('Manage Workflows') }}</h1>
         </ng-include>
       </div>
       <button
-        ng-click="refreshWorkflowsList()"
+        ng-click="refreshWorkflowsList(true)"
```

Comment:
Functions with boolean parameters are a bit confusing because it's difficult to infer what the parameter does. Unless it's something like `switchLamp(true)` or similar (and I'd even debate that)

Can we fix this in one of these solutions:

1 - pass the parameter as an object:
```js
refreshWorkflowsList({ resetPagination: true });
```

2 - split the function in two:
```js
resetPagination(); // this would have `$scope.pageObj = { total: 0, size: 25, num: 1 };`
refreshWorkflowsList();
```

Option two would be better because it's clear what every function does, and we don't have a single function that does a lot, but I think it's a bit difficult to split functions like this in AngularJS because calling them from the template might be a bit awkward. I don't remember if you can do something like:

```html
<div
  ng-click="resetPagination(); refreshWorkflowsList();"
></div>
```